rpcs3/rpcs3qt/GameCompatibility.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <QJsonDocument>
+#include <QJsonObject>
+#include <QtNetwork/QNetworkAccessManager>
+#include <QtNetwork/QNetworkReply>
+#include <QtNetwork/QNetworkRequest>
+#include <QTableWidget>
+
+class GameCompatibility
+{
+private:
+	int compatibilityColumn;
+	int serialColumn;
+	QNetworkAccessManager *networkAccessManager;
+	QTableWidget *tableWidget;
+	const std::map<QString, QString> tooltips = {
+		{ "Playable", QObject::tr("Games that can be properly played from start to finish") },
+		{ "Ingame", QObject::tr("Games that go somewhere but not far enough to be considered playable") },
+		{ "Intro", QObject::tr("Games that only display some screens") },
+		{ "Loadable", QObject::tr("Games that only display some screens") },
```

Comment:
Intro and Loadable have the same description. It should be **_Games that display a black screen with an active framerate_** for Loadable status.